libr/asm/arch/arc/gnu/arcompact-dis.c

```diff
@@ -38,6 +38,9 @@
     if dbg is 1 then this definition is required
```

Comment:
modifying this file like this means that we will not be able to update the code from binutils anymore